source/draw/RasterYuv420Semiplanar.ooc

```diff
@@ -247,4 +247,12 @@ RasterYuv420Semiplanar: class extends RasterYuvSemiplanar {
 		fileWriter close()
 	}
 	createPaintEngine: override func -> PaintEngine { Yuv420PaintEngine new(this) }
+	kean_draw_rasterYuv420Semiplanar_new: static unmangled func (width, height, stride: Int, monochromeData: Void*, uvData: Void*) -> This {
```

Comment:
Fixed, but I created another commit as I don't know if I can amend the last one (PR merged)